server/handlers/delete_account_oauth_test.go

```diff
@@ -0,0 +1,66 @@
+package handlers_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/keratin/authn-server/lib/route"
+	"github.com/keratin/authn-server/server/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeleteAccountOauth(t *testing.T) {
+	app := test.App()
+
+	server := test.Server(app)
+	defer server.Close()
+
+	client := route.NewClient(server.URL).Authenticated(app.Config.AuthUsername, app.Config.AuthPassword)
+
+	http.DefaultClient.CheckRedirect = func(req *http.Request, via []*http.Request) error {
+		return http.ErrUseLastResponse
+	}
+
+	t.Run("delete social account", func(t *testing.T) {
+		account, err := app.AccountStore.Create("deleted-social-account@keratin.tech", []byte("password"))
+		require.NoError(t, err)
+
+		time.Sleep(5 * time.Second)
+
+		err = app.AccountStore.AddOauthAccount(account.ID, "test", "DELETEDID", "email", "TOKEN")
+		require.NoError(t, err)
+
+		payload := map[string]interface{}{"oauth_providers": []string{"test"}}
+		res, err := client.DeleteJSON("/accounts/1/oauth", payload)
+		require.NoError(t, err)
+
+		require.Equal(t, http.StatusOK, res.StatusCode)
+		require.Equal(t, []byte{}, test.ReadBody(res))
+	})
+
+	t.Run("return not found when user does not exists", func(t *testing.T) {
```

Comment:
"user does not exist" - generally shorter test names will read better in output on failures.  And the assertions should take care of the "return not found" stuff for you.  It is frustrating when the assertions change but the test names don't.